tox.ini

```diff
@@ -1,49 +1,56 @@
 [tox]
 minversion = 3.5.0
-skipdist = True
+skipsdist = True
 skip_missing_interpreters = True
 isolated_build = True
+toxworkdir = {env:TOXWORKDIR:/tmp/{env:USER}/tox}
 envlist = lint,server,agent
 
 [testenv]
 usedevelop = True
-install_command = pip install -U {opts} {packages}
+install_command = pip install --no-cache-dir --progress-bar off --prefix={envdir} {opts} {packages}
+passenv =
+    PY_COLORS
+    NO_COLORS
+    TERM
+    USER
 setenv =
-   VIRTUAL_ENV={envdir}
+    VIRTUAL_ENV = {envdir}
+    XDG_CACHE_HOME = {envdir}
+    SKIP_GENERATE_AUTHORS = 1
+    SKIP_WRITE_GIT_CHANGELOG = 1
```

Comment:
I believe that these are supposed to quiet PBR, but evidently they don't work.

Reply:
Bummer.  Originally they did work, but maybe there is something about the `tox --current-env` that defeats them.